week-1/D-variables/exercise.js

```diff
@@ -1,3 +1,8 @@
 // Start by creating a variable `greeting`
 
+//Print hellow world 3 times
+
+var greeting = 'Hello World';
+console.log(greeting);
 console.log(greeting);
+console.log(greeting);
```

Comment:
Good. The usual way to repeat like this would be to use a loop construct but this hasn't been covered yet.

Reply:
great.. I have just modified it.
thanks